arxiv/base/templates/base/macros.html

```diff
@@ -261,7 +261,7 @@ <h1 class="title mathjax"><span class="descriptor">Title:</span>{{ title|tex2utf
 
 Title: {{ title }}
 Authors: {{ authors }}
-Categories: {{ primary_category }}{%- for category in secondary_categories|sort(attribute='id') -%} {{ category }}{%- endfor %}
+Categories: {{ primary_category.id }}{%- for category in secondary_categories|sort(attribute='id') -%} {{ category }}{%- endfor %}
```

Comment:
I wonder if we should define `__str__` for Category so this could be `{{primary_category}}` ?

Reply:
I'm not sure it has a couple string methods - notably display() which creates a longer string meant for display purposes. There is also full_name